consensus.go

```diff
@@ -501,17 +511,35 @@ func (p *Pbft) runValidateState(ctx context.Context) { // start new round
 			panic(fmt.Errorf("BUG: Unexpected message type: %s in %s", msg.Type, p.getState()))
 		}
 
-		if p.state.numPrepared() > p.state.NumValid() {
-			// we have received enough prepare messages
-			sendCommit(span)
-		}
+		if p.config.VotingPower == nil {
```

Comment:
There is a possibility not to use VotingPower, and go the old way?

Reply:
Should we add some info here so that we know that if not set default is used?

Reply:
The idea was to be able to select counting and voting power algorithms. I'm not sure that i understood your question @Nemanja0x.
@0xSasaPrsic Added comments

Reply:
"The idea was to be able to select counting and voting power algorithms" - this is the answer. :)